test_scripts/API/AddCommand/ATF_AddCommand_VR_AddCommand_WARNINGS.lua

```diff
@@ -0,0 +1,137 @@
+-- 2. Performed steps: 
+-- MOB -> SDL: sends AddCommand
+-- SDL -> HMI: resends VR.AddCommand with imageType of cmdIcon which is not supported by HMI and UI.AddCommand with all valid params
+-- HMI -> SDL: VR.AddCommand (WARNINGS), UI.AddCommand (SUCCESS)
+--
+-- Expected result:
+-- SDL -> MOB: appID: (WARNINGS, success: true: AddCommand)
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+
+--[[ Local Variables ]]
+local storagePath = config.pathToSDL .. "storage/"
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()			
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications["Test Application"]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if
+    data.result.isSDLAllowed ~= true then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)						
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = "127.0.0.1"}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,_)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+        :Times(2)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+function Test:Precondition_PutFile()
+  local cid = self.mobileSession:SendRPC("PutFile",
+  {			
+    syncFileName = "icon.png",
+    fileType	= "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  }, "files/icon.png")
+  
+  EXPECT_RESPONSE(cid, { success = true})			
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_AddCommand_VR_warning()
+  local cid = self.mobileSession:SendRPC("AddCommand",
+  {
+    cmdID = 11,
+    menuParams = 	
+    { 
+      parentID = 1,
+      position = 0,
+      menuName ="Commandpositive"
+    }, 
+    vrCommands = 
+    { 
+      "VRCommandonepositive",
+      "VRCommandonepositivedouble"
+    }, 
+    cmdIcon = 	
+    { 
+      value ="icon.png",
+      imageType ="DYNAMIC"
+    }
+  })
+  EXPECT_HMICALL("UI.AddCommand", 
+  { 
+    cmdID = 11,
+    cmdIcon = 
+    {
+      value = storagePath .."icon.png", 
```

Comment:
@vvvakulenko,  SDL can return absolute path for this and you should check that value is sub-string of the full SDL path that is returned.
Please make verification of 'value' to be checked as sub-string, because at the moment the test has an error:
Exp: ./SDL_bin/storage/0000001_12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0/icon.png
Real: /home/istoimenova/SmartDeviceLinkCore/test_run/SDL_bin/storage/0000001_12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0/icon.png

You can use as example:
if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
-- print some error
end